### PR TITLE
Redesign Audit Logs PHP Example Application

### PR DESCRIPTION
**Pull request by kmazariegos**
Redesign Audit Logs PHP Example Application

-Changed UI and some of the functionality
-Removed old events file

### CODE REVIEW

**Review by awolfden — CHANGES_REQUESTED** [review]
Lookin good, just a few things to adjust

File: php-audit-logs-example/templates/login.html.twig [hunk 1]
```diff
@@ -1,28 +1,124 @@
...
-    <div class='flex_column'> 
-        <div class="flex heading_div">
-            <img src="workos-logo-with-text.png" alt="workos logo"> 
+<body class="container_success">
+    <div class="logged_in_div_left flex_column">
+        <div class="flex logged_in_div_left_title">
+            <div class="flex-column">
+                <div class="flex space-between">
+                    <div><h2>Audit Logs</h2></div>
+                    <div><img src="settings.png"></div>
```
[the remark this answers is not in the document]
Comment (kmazariegos) [reply php-audit-logs-example/templates/login.html.twig]: Done! thank you!

File: php-audit-logs-example/templates/send_events.html.twig [hunk 2]
```diff
@@ -1,103 +1,212 @@
...
+                                            <div><label for="filter-actors">Actors</label></div>
+                                            <div><input name="filter-actors" class="text-input" id="filter-actors" type="text" placeholder="Optional comma-separated list of actors to filter on"></div>
+                                        </div> 
+                                        <div class="event-form-input-group flex-column flex-start">
+                                            <div><label for="filter-targets">Targets</label></div>
+                                            <div><input  name="filter-targets" class="text-input" id="filter-targets" type="text" placeholder="Optional comma-separated list of targets to filter on"></div>
+                                        </div> 
+                                    </div>
+                                    <div class="flex flex-start">                                    
+                                        <div class="flex width-150px">
```
[the remark this answers is not in the document]
Comment (kmazariegos) [reply php-audit-logs-example/templates/send_events.html.twig]: Done!

File: php-audit-logs-example/router.php [hunk 3]
```diff
@@ -118,15 +154,6 @@ function objectToArray($d)
...
-    case ("/export_events"):
-        session_start();
-        $payload = file_get_contents("php://input");
-        $orgId = $_SESSION['id'];
-        $orgName = $_SESSION['name'];
-        echo $twig->render("export_events.html.twig", ['org_id' => $orgId, 'org_name' => $orgName]); 
-        return true;
-
 //generate_csv
     case ("/get_events"):
```
[the remark this answers is not in the document]
Comment (kmazariegos) [reply php-audit-logs-example/router.php:158]: Added comment to add username in path. wondering if that was the issue?

**Review by awolfden — CHANGES_REQUESTED** [review]
For the file path, it looks like we just need the username of the system user to generate the path to the downloads folder. Could we add an environment variable where the user adds their username and then build the file path off of that automatically, and then add that as a step in the Readme next to where the other env variables are set? 

It's not super obvious that the user would need to go into the code and update the username so I think we should make this more visible.

File: php-audit-logs-example/templates/send_events.html.twig [hunk 4]
```diff
@@ -1,103 +1,211 @@
...
+                                            <div><label for="filter-actors">Actors</label></div>
+                                            <div><input name="filter-actors" class="text-input" id="filter-actors" type="text" placeholder="Optional comma-separated list of actors to filter on"></div>
+                                        </div> 
+                                        <div class="event-form-input-group flex-column flex-start">
+                                            <div><label for="filter-targets">Targets</label></div>
+                                            <div><input  name="filter-targets" class="text-input" id="filter-targets" type="text" placeholder="Optional comma-separated list of targets to filter on"></div>
+                                        </div> 
+                                    </div>
+                                    <div class="flex flex-start">                                    
+                                        <div>
```
[the remark this answers is not in the document]
Comment (kmazariegos) [reply php-audit-logs-example/templates/send_events.html.twig:158]: Great idea! i'll do that now :)

**Review by awolfden — APPROVED** [review]
🤙

**Pull request merged**